atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/TransactionManagerTest.java

```diff
@@ -49,6 +50,26 @@
 import com.palantir.timestamp.TimestampService;
 
 public class TransactionManagerTest extends TransactionTestSetup {
```

Comment:
*weep, `TransactionTestSetup` could have easily been a rule/external resource. Instead we have 2 layers of abstract tests 😭

Reply:
Had to stop refactoring somewhere